bolt/lib/Profile/YAMLProfileReader.cpp

```diff
@@ -479,6 +481,11 @@ Error YAMLProfileReader::readProfile(BinaryContext &BC) {
   NormalizeByInsnCount = usesEvent("cycles") || usesEvent("instructions");
   NormalizeByCalls = usesEvent("branches");
 
+  // Map profiled function ids to names.
+  DenseMap<uint32_t, std::string *> IdToFunctionName;
```

Comment:
Do we need a map and not a vector?

Reply:
You're right, a vector is a better choice here